src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -624,81 +624,6 @@
     "src": "/compensation/resources_comp0218.asp",
     "dest": "/disability/compensation-rates/special-monthly-compensation-rates/"
   },
-  {
```

Comment:
**Second chunk**
https://www.benefits.va.gov/COMPENSATION/resources_comp0214.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0213.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0212.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0211.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0209.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0208.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0207.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0206.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0205.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0204.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0203.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0202.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0201.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0200.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0299.asp 

**Note: DON'T do anything with these (leave as is with redirects):**
https://www.benefits.va.gov/COMPENSATION/resources_comp02.asp 
https://www.benefits.va.gov/COMPENSATION/resources_comp0218.asp